server-spi/src/main/java/org/keycloak/models/OTPPolicy.java

```diff
@@ -18,6 +18,7 @@
 package org.keycloak.models;
 
 import org.jboss.logging.Logger;
+import org.keycloak.authentication.otp.OTPApplicationProvider;
```

Comment:
Updated

Reply:
@stianst Thanks. WDYT about my question above ( `server-spi` VS `server-spi-private` )?

Reply:
Was a bit half and half on where to put it. On the one side I figured server-spi as it's a small/simple SPI that would be useful to folks until we support JSON variants (honestly personally I'd probably just do a Java provider myself though), on the other side we do tend to put things into server-spi-private (and not support any custom providers ;)).

Moved it to server-spi-private for now, as there's plenty of folks using these anyways. I do think we may want to change our policies around SPIs soon though. Perhaps we should make more "public" and instead have a policy that says we won't always make them backwards compatible, but rather may break the APIs (and provide migration docs), which since we're always pushing new major versions of Keycloak is fine from a semver perspective, we just need to make sure we communicate the fact that we reserve the right to change the APIs in a breaking way (and document it properly when we do).

Reply:
+1 to change the policy this way. Thanks for the changes.